ebusd-2.1.x/de/encon/3c.Excellent300.csv

```diff
@@ -0,0 +1,159 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,field2,part (m/s),datatypes/templates,divider/values,unit,comment,field3,part (m/s),datatypes/templates,divider/values,unit,comment,field4,part (m/s),datatypes/templates,divider/values,unit,comment,field5,part (m/s),datatypes/templates,divider/values,unit,comment
+## Diese Ebus-Konfiguration könnte für Ubbink, VisionAIR, WOLF CWL Serien, Viessmann und einige andere Systeme funktionieren, die nur umgebrandete Brink-Geräte sind
+## Quellen:
+## - Ursprüngliche Idee und einige Trennzeichen: https://github.com/dstrigl/ebusd-config-brink-renovent-excellent-300
+## - Brink Service Tool (decompiliert via Jetbrains DotPeak): https://www.brinkclimatesystems.nl/tools/software-brink-service-tool-en
+## - Renovent 150 Datenblatt: https://manuals.plus/brink/renovent-sky-150-plus-mechanical-ventilation-with-heat-recovery-manual
+## - Modbus Modul Datenblatt: https://www.brinkclimatesystems.nl/documenten/modbus-uwa2-b-uwa2-e-installation-regulations-614882.pdf
+## Nachrichtennamen basieren auf den offiziellen Übersetzungen des Brink Service Tools mit entfernten Leerzeichen und Sonderzeichen.
+## Nachrichtenkommentar ist der Name des Parameters, wie er intern im Code verwendet wird (um zu helfen, wenn die Übersetzung selbst verwirrend ist)
+
+*r,Excellent300,,,,3c,
+*w,Excellent300,,,,3c,
+
+## ALLGEMEINE HRU BEFEHLE ## (WTWCommands.cs - Einige davon sind möglicherweise nicht für dieses Gerät anwendbar, mit Vorsicht verwenden)
+w,,RücksetzenAufWerkseinstellung,FactoryReset,,,40ff,466163746f72795265736574
```

Comment:
I guess the files are generated, so please make the message and field names identical in en+de, which is the base for translation. maybe the circuit+name fields were accidentially exchanged in the de files?

Reply:
The variable names are taked 1:1 from the official Brink translations in the Brink Service Tool. The comments are the variables names from the C# code of the tool for easier refenrecing. If you believe the comments are not helpful, I can drop those.

Or am I missing the point of your concern @john30?

Reply:
it should be the other way around, i.e. message names are expected to be identical in both languages for one message whereas comments are expected to be different.
e.g. FanMode should have the de comment "Ventilatorbetrieb" and en "FanMode"

Reply:
Oh... my bad! To be honest I only studied the EN files (not the smartest decision in the hindsight), and for whatever reason blindly assumed this is the way to do it - I think the comments are not really used in Home Assistant, only the variable names, but for the sake of consistency I totally agree to make it consistent with the rest of the config files. Should be fixed now.